client/galaxy/scripts/nls/fr/locale.js

```diff
@@ -6,61 +6,61 @@ define({
 "Workflow":
     "Workflow",
 "Shared Data":
-    "Partage de données",
+    "Données partagées",
 "Data Libraries":
-    false,
+    "Bibliothèque de données",
 "Histories":
-    false,
+    "Historiques",
 "Workflows":
-    false,
+    "Workflows",
 "Visualizations":
-    false,
+    "Visualisations",
 "Pages":
-    false,
+    "Pages",
 "Visualization":
-    "Représentation des données",
+    "Visualisation",
 "New Track Browser":
-    false,
+    "Nouveau Navigateur de Tracks/Pistes",
 "Saved Visualizations":
-    false,
+    "Visualisations sauvegardés",
 "Interactive Environments":
-    false,
+    "Environnements interactifs",
```

Comment:
Not sure we have to make a direct word by word translation....  but I don't have better proposition.